xbmc/windowing/X11/VideoSyncOML.h

```diff
@@ -28,6 +36,9 @@ class CVideoSyncOML : public CVideoSync, IDispResource
 
 private:
   std::atomic_bool m_abort;
-  CWinSystemX11GLContext &m_winSystem;
+  CWinSystemX11GLContext* m_winSystem;
```

Comment:
I prefer `CWinSystemX11GLContext& m_winSystem` for safety against pointer dereferences. If polymorphism is what you're after, both `*` and `&` hit the vtable. If `*` is preferred, then deference safety can be increased with:

```c++
CWinSystemX11GLContext* const m_winSystem;
```